### ISSUE TITLE
External group

### ISSUE THREAD

**Issue by scemama** [TREX-CoE/trexio#112]
https://github.com/psi4/psi4/issues/2847#issuecomment-1410051003 
>  We could probably add a functionality allowing to write an arbitrary variable in e.g. "external" group via generic trexio_write|read_(file, variable-str, datatype-str, size-max). I can implement it easily for the HDF5 back end but TEXT one is more tricky.

**Comment by scemama** [comment]
For the text back-end, we can search for the name of the attribute as usual. If it is difficult, a simple workaround would be to create one file for each external data. This way, the name of the external data would be the name of the text file.

**Comment by q-posev** [comment]
Yes, one text file per attribute is the easiest way to implement this. I will come back to this when I have more time.
